template/api/docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: "3.3"
```

Comment:
Consider generating this in the makefile, to avoid specifying the zenkit-build image in many places.